README.md

```diff
@@ -22,6 +23,17 @@ Welcome to Bridge-To-Kubernetes! Bridge to Kubernetes extends the Kubernetes per
 - [Visual Studio](https://docs.microsoft.com/en-us/visualstudio/bridge/bridge-to-kubernetes-vs)
 - [Visual Studio Code](https://docs.microsoft.com/en-us/visualstudio/bridge/bridge-to-kubernetes-vs-codeart)
 
+## CLI tool installation

```

Comment:
@qpetraroia  and @sabbour Please review , this is documentation update for CLI tool installation instructions. Thank you.